src/frontend/komponenter/TilordnetSaksbehandler/TilordnetSaksbehandlerVenstremeny.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+
+import { Alert, BodyShort, HStack } from '@navikt/ds-react';
+import { AGray50, AWhite } from '@navikt/ds-tokens/dist/tokens';
+
+import { utledVisningsnavn } from './tilordnetSaksbehandlerUtils';
+import { useBehandling } from '../../context/BehandlingContext';
+import { BehandlingType } from '../../typer/behandling/behandlingType';
+import { SaksbehandlerRolle } from '../../typer/behandling/saksbehandlerDto';
+
+const TilordnetSaksbehandlerVenstremeny: React.FC = () => {
+    const { behandling } = useBehandling();
+
+    if (!behandling.tilordnetSaksbehandler) {
+        return;
+    }
+
+    return (
+        <>
+            {behandling.tilordnetSaksbehandler?.rolle !==
+                SaksbehandlerRolle.OPPGAVE_FINNES_IKKE && (
+                <HStack
+                    gap={'2'}
+                    align={'center'}
+                    paddingInline={'space-16 space-0'}
+                    paddingBlock={'space-8 space-0'}
```

Comment:
Kan styling her + bakgrunnsfarge styres av en container som er rundt både denne og oppsummeringkomponenten? Så slipper vi å ha to steder som setter samme farge + spacing styres utenfra og ikke av hver enkelt